### PR TITLE
Style/#229 landing page hero section css

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
Five Landing Page Hero section breakpoint implementation.

They are store as variable in the src/styles/_variable.scss file.

Screenshot 2023-08-02 at 9 51 32 PM

Please download this branch or use the deploy link at the bottom to test the css layout of these five break points.

This branch contain updates for the rest of Patrick's feedback in Zenhub:
https://app.zenhub.com/workspaces/overture-stack-5d2e058ff67cc800011fee6b/issues/gh/overture-stack/website/229

### CODE REVIEW

File: src/components/Button/index.js [hunk 1]
```diff
@@ -4,8 +4,9 @@ import { AnchorLink } from 'gatsby-plugin-anchor-links';
 import './styles.scss';
 
 const btnTypes = {
-  primary: 'is-primary', // uses bulma
-  secondary: 'is-white secondary', // bulma + custom css
+  primary: 'button is-primary', // uses bulma
+  secondary: 'button is-white secondary', // bulma + custom css
+  plain: 'plain',
 };
```
Comment (daniel-cy-lu) [inline src/components/Button/index.js:10]: found we need button with no hover state, plain type is added

File: src/components/NavBar/MegaMenu/index.js [hunk 2]
```diff
@@ -2,9 +2,7 @@
  * Component: Display the Megamenu on mouse over / tap.
  */
 import React from 'react';
-import productsDict from 'constants/products';
-import { PRODUCTS_PATH } from 'constants/pages';
-import { Badge, ComingSoonBadge, Icon, IconCommon, LinkHelper as Link } from 'components';
+import { Badge, ComingSoonBadge, Icon, LinkHelper as Link } from 'components';
```
Comment (daniel-cy-lu) [inline src/components/NavBar/MegaMenu/index.js:5]: MegaMenu changes are for #6 of partrick's feedback

File: src/components/NavBar/MegaMenuLink/index.js [hunk 3]
```diff
@@ -19,7 +19,13 @@ const MegaMenuLink = ({ children, isActive, name, path, toggleMegaMenu, type })
         >
           {name}
           {name === 'Documentation' ? (
```
Comment (daniel-cy-lu) [inline src/components/NavBar/MegaMenuLink/index.js:21]: This is for #7 of Patrick's feedback

File: src/components/Typography/styles.scss [hunk 4]
```diff
@@ -11,8 +11,8 @@
   line-height: 70px;
```
Comment (daniel-cy-lu) [inline src/components/Typography/styles.scss:11]: Add smaller font size for mobile breakpoint to match mock up

File: src/components/Button/index.js (hunk 1 above)
Comment (samrichca) [reply src/components/Button/index.js:10]: a better name for this is `default`

File: src/components/NavBar/MegaMenu/index.js (hunk 2 above)
Comment (samrichca) [reply src/components/NavBar/MegaMenu/index.js:5]: please link this PR to feedback ticket

File: src/pages/homepagestyles.scss [hunk 5]
```diff
@@ -0,0 +1,81 @@
+.overture-image-section {
```
Comment (samrichca) [inline src/pages/homepagestyles.scss:1]: please delete the temp homepage image above ^

**Comment by samrichca** [comment]
this looks great! huge improvement! I had a couple of notes, will approve when they're resolved

File: src/components/Button/index.js (hunk 1 above)
Comment (daniel-cy-lu) [reply src/components/Button/index.js:10]: I will update it! :)

File: src/components/NavBar/MegaMenu/index.js (hunk 2 above)
Comment (daniel-cy-lu) [reply src/components/NavBar/MegaMenu/index.js:5]: issue is linked

File: src/pages/homepagestyles.scss (hunk 5 above)
Comment (daniel-cy-lu) [reply src/pages/homepagestyles.scss:1]: scss file removed

**Comment by daniel-cy-lu** [comment]
> this looks great! huge improvement! I had a couple of notes, will approve when they're resolved

Sam, I resolved all but removal of the temporary scss file. I have it removed in the next sections and updated the className.

**Review by samrichca — APPROVED** [review]

**Pull request merged**